online_catalog/TRLN-Discovery/ICPSR/profile.txt

```diff
@@ -0,0 +1,242 @@
+Statistical Elements Analysis of 10721 xml documents with 241 elements

```

Comment:
This is the result of running the new xml-analyzer script on the whole ICPSR record set. Not sure if it's of any use to you, but I'd wanted quick-and-easy way of getting this kind of data about a record set for a while. 

Not entirely sure exactly all it's spitting out (the number in brackets confuses me a little), but: 
* each unique node in record set with full path to node
* weird indication of occurrences/cardinality of that node within each record(?)
* % of records using that node